documentation/modules/ROOT/pages/connectors/vitess.adoc

```diff
@@ -773,7 +773,7 @@ a|_n/a_
 a|_n/a_
 
 |`JSON`
-|Unsupported yet
+|`STRING`
```

Comment:
I suppose it is exported as the `io.debezium.data.Json` logical type?

Reply:
That's a good point. Unfortunately, Vitess Connector doesn't export any `io.debezium` semantic type (schema name) yet, same for other column types. 

I'll try and add `io.debezium.data.Json` semantic type (schema name) to the `JSON` column type in the corresponding PR: https://github.com/debezium/debezium-connector-vitess/pull/19